src/Line.js

```diff
@@ -540,7 +540,7 @@ export function Line(opts, data) {
 		series.forEach((s, i) => {
 			if (i > 0 && s.show) {
                 drawPath(i);
-                fire("drawSeries", s, i);
+                fire("drawSeries", i-1);
```

Comment:
right now i'm passing the raw internal `i` here [1], so this should not be adjusted to assume differently. whether this is ultimately a good decision is probably another topic, but let's keep it uniform in this PR.

[1] https://github.com/leeoniya/uPlot/blob/master/src/Line.js#L929

Reply:
i've opened #77 to figure this out.